src/DonationForms/resources/components/Pagination.module.scss

```diff
@@ -0,0 +1,74 @@
+:global(#give-admin-donation-forms-root) {
```

Comment:
I’m not going to look too much in the styles, because I will need to change a few things when I merge this into my branch. Looks good otherwise.